include/solvers/heat_transfer_scratch_data.h

```diff
@@ -99,13 +98,15 @@ class HeatTransferScratchData
                           const Quadrature<dim>          &quadrature,
                           const Mapping<dim>             &mapping,
                           const FiniteElement<dim>       &fe_fd,
-                          const Quadrature<dim - 1>      &face_quadrature)
+                          const Quadrature<dim - 1>      &face_quadrature,
+                          const double                    delta_T_ref)
     : properties_manager(properties_manager)
     , fe_values_T(mapping,
                   fe_ht,
                   quadrature,
                   update_values | update_quadrature_points | update_JxW_values |
                     update_gradients | update_hessians)
+    , global_delta_T_ref(delta_T_ref)
```

Comment:
I would add an assertion in the constructor to check if that value is >0 just to be defensive. You never know and this cost nothing